app/(auth)/login/login-form.jsx

```
@@ -123,49 +146,51 @@
+                    </md-outlined-text-field>
+                    <md-outlined-text-field
+                        error={state.errors.password || state.errors.general}
+                        label="Mật khẩu"
+                        placeholder="Nhập mật khẩu"
+                        value={state.formData.password}
+                        type={revealedPassword ? "text" : "password"}
+                        className="max-w-[560px] w-[80vw]"
+                        onInput={handlePasswordChange}
+                    >
+                        <md-icon slot="leading-icon">password</md-icon>
+                        <md-icon slot="trailing-icon" onClick={() => setRevealedPassword(!revealedPassword)}>{revealedPassword ? "visibility" : "visibility_off"}</md-icon>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)